src/test/moment/duration.js

```diff
@@ -53,10 +53,17 @@ test('milliseconds instantiation', function (assert) {
 
 test('undefined instantiation', function (assert) {
     assert.equal(moment.duration(undefined).milliseconds(), 0, 'milliseconds');
+    assert.equal(moment.duration(undefined)._isValid, true, '_isValid');
```

Comment:
This needs an `isValid()` accessor function. We  don't want people checking underscored properties.